src/main/java/domain/Board.java

```diff
@@ -113,4 +117,22 @@ public Team determineWinTeam() {
         }
         return Team.RED;
     }
+
+    public Map<PieceType, Integer> countPieces(final Team team) {
+        if (team == Team.GREEN) {
```

Comment:
이미 Team Enum 에 isGreenTeam, isRedTeam 메서드를 만들어 두셨는데 이를 활용해보면 좋지 않을까요?

Reply:
1차에서 피드백 주신 내용으로 리팩터링을 했는데 해당 부분 로직을 교체하지 않은 것 같습니다.
알려주셔서 감사합니다.  
해당 부분을 아래처럼 변경했습니다.

```suggestion
        if (team.isGreenTeam()) {

    }
```